### PR TITLE
i added full log in form

### PR DESCRIPTION
**Pull request by Ashakour1**

### CODE REVIEW

File: package.json
Comment (duraanali) [inline package.json:1]: Asc Abdishakour, waa ku mahadsantahay code-ka, waxaan rabay inaan ogaado maxaa Tailwind devDependencies u galisay?

Comment (Ashakour1) [reply package.json:1]: wcs eng cafwan  wakabaxiyay sidokle code oo dhamestiran aya hada Kuso push gareynaya wixii Kaqaldan ilasowadaag mahadsan

File: package.json [hunk 1]
```diff
@@ -20,7 +20,6 @@
     "postcss": "8.4.27",
     "react": "18.2.0",
     "react-dom": "18.2.0",
-    "tailwindcss": "3.3.3",
```
Comment (duraanali) [inline package.json:23]: Tailwindcss aa ka bixisay uu leeyahay!

Comment (Ashakour1) [reply package.json:23]: ok waan soxalina wait

**Review by duraanali — APPROVED** [review]
Looks good to me, Great work, thanks.

**Pull request merged**